IntegrationTests/PurchaseTester/PurchaseTester/SwiftPaywall.swift

```diff
@@ -17,7 +17,7 @@ enum PayWallEdgeStyle : String {
 }
 
 @objc protocol SwiftPaywallDelegate {
-    func purchaseCompleted(paywall: SwiftPaywall, transaction: SKPaymentTransaction, customerInfo: CustomerInfo)
+    func purchaseCompleted(paywall: SwiftPaywall, transaction: StoreTransaction, customerInfo: CustomerInfo)
```

Comment:
this delegate is unused, but I figured I might as well keep it and just update the type